### PR TITLE
ft user view and edit profile

### PR DESCRIPTION
**Pull request by emilenfc**
# Description

user view and edit account

Fixes # (issue)

## Type of change

Please check the options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature or Enhancement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have created a loom video for the new feature or enhancement
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/features/profile/index.ts
Comment (mucyoalexaxel) [inline src/features/profile/index.ts:1]: @emilenfc Delete this file

File: src/features/profile/sevices/GetUser.tsx
Comment (mucyoalexaxel) [inline src/features/profile/sevices/GetUser.tsx:1]: @emilenfc Why this ?

**Review by mucyoalexaxel — CHANGES_REQUESTED** [review]
@emilenfc Check suggestions below

File: src/features/profile/sevices/GetUser.tsx
Comment (emilenfc) [reply src/features/profile/sevices/GetUser.tsx:1]: I didn't use it

**Comment by shyakadavis** [comment]
I am trying to update the profile of  a user and this is what I am getting.

Screenshot 2023-05-08 at 21 19 21

**Comment by shyakadavis** [comment]
What is wrong @emilenfc ?

**Review by musllim — DISMISSED** [review]
LGTM

**Review by musllim — APPROVED** [review]
LGTM

**Review by mucyoalexaxel — APPROVED** [review]

**Pull request merged**